### DISCUSSION TITLE
superset/sql_json/ 405 method Not allowed

### DISCUSSION THREAD

**Discussion by telmofranciscmultivision** [apache/superset#13985] [Q&A / Help]
Hi, I'm trying to integrate superset with amundsen but without good news.
While trying to GET localhost:8088/superset/sql_json/
`{database: "mysql", schema: "movies_db", tableName: "persons", cluster: "def"}`

Response:
`<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>405 Method Not Allowed</title>
<h1>Method Not Allowed</h1>
<p>The method is not allowed for the requested URL.</p>`

Any help?

Best Regards

**Comment by maltoze** [comment]
https://github.com/apache/superset/blob/ab0f5e3087949d9382765020a44e33aa684a476f/superset/views/core.py#L2444

`/sql_json/` only support `POST`
